enterprise/backend/test/metabase_enterprise/advanced_permissions/driver/impersonation_test.clj

```diff
@@ -95,15 +96,37 @@
     (premium-features-test/with-premium-features #{:advanced-permissions}
       (advanced-perms.api.tu/with-impersonations {:impersonations [{:db-id (mt/id) :attribute "impersonation_attr"}]
                                                   :attributes     {"impersonation_attr" "limited_role"}}
-        ;; User with connection impersonation should not be able to query a table they don't have access to
-        ;; (`limited_role` in CI Snowflake has no data access)
-        (is (thrown-with-msg? clojure.lang.ExceptionInfo
-                              #"SQL compilation error:\nDatabase.*does not exist or not authorized"
-                              (mt/run-mbql-query venues
-                                                 {:aggregation [[:count]]})))
-        ;; Non-impersonated user should stil be able to query the table
+        ;; Test database initially has no default role set. All queries should fail, even for non-impersonated users,
```

Comment:
Is this true for every db driver? I thought part of this was that some drivers _do_ have default roles that work without configuring them. So, would this part of the test fail for one of those dbs now?

Reply:
This test is only for Snowflake! Postgres (the only other driver that can use conn impersonation at the moment) has a separate test above

Reply:
Ah, oops. Sorry for the noise. Just missed it my bad.